semantic_release/history/logs.py

```diff
@@ -173,7 +173,14 @@ def markdown_changelog(version: str, changelog: dict, header: bool = False) -> s
         # Add a heading with the version number
         output += '## v{0}\n'.format(version)
 
-    for section in CHANGELOG_SECTIONS:
+    custom_changelog_sections = config.get("semantic_release", "changelog_sections", fallback="")
+    if custom_changelog_sections:
+        debug('markdown_changelog(custom_changelog={})'.format(custom_changelog_sections))
+        changelog_sections = [s.strip() for s in custom_changelog_sections.split(',')]
+    else:
+        changelog_sections = DEFAULT_CHANGELOG_SECTIONS
+
+    for section in changelog_sections:
         if not changelog[section]:
```

Comment:
Maybe add a check to skip invalid sections:

```suggestion
        if section not in changelog or not changelog[section]:
```

Reply:
Will do